aws-lambda-java-serialization/RELEASE.CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### November 21, 2022
+`1.0.1`:
+- Updated Jackson dependencies
```

Comment:
Could you include the version number in the change long

Reply:
updated